Sources/StreamVideoSwiftUI/CallView/LayoutComponents/BottomParticipantsBarLayoutComponent.swift

```diff
@@ -0,0 +1,114 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamVideo
+import SwiftUI
+
+/// `BottomParticipantsBarLayoutComponent` represents a horizontally scrollable view of participant thumbnails.
+/// This component lays out participant thumbnails in a bar at the bottom of the associated view.
+public struct BottomParticipantsBarLayoutComponent<Factory: ViewFactory>: View {
+
+    // MARK: - Properties
+
+    /// Factory for creating views.
+    public var viewFactory: Factory
+
+    /// List of participants to display.
+    public var participants: [CallParticipant]
+
+    /// Frame in which the component will be laid out.
+    public var frame: CGRect
+
+    /// Information about the call (if available).
+    public var call: Call?
+
+    /// Size of each participant thumbnail.
+    public var thumbnailSize: CGFloat
+
+    /// Flag to determine if all participant information should be shown.
+    public var showAllInfo: Bool
+
+    /// Closure called to change visibility of a participant's track.
+    public var onChangeTrackVisibility: @MainActor(CallParticipant, Bool) -> Void
```

Comment:
I was considering the same but the layouts have the same structure (both the call and the onChangeTrackVisibility). I was thinking that may be on purpose in order to allow further customisation.

Reply:
We can control this only via the call object (since the participants state is there). Therefore, we can safely simplify the API here and remove this property.